<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Files</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item>
     <widget class="QLabel" name="labelStatus">
      <property name="text">
       <string>Ready.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>494</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionUpdate_Database"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="checkBoxCaseSensitive"/>
    <addaction name="checkBoxRegExp"/>
    <addaction name="checkBoxSearchOnlyHome"/>
    <addaction name="checkBoxShowFullPath"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <action name="actionUpdate_Database">
   <property name="text">
    <string>Update Database</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="checkBoxCaseSensitive">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Case Sensitive</string>
   </property>
  </action>
  <action name="checkBoxRegExp">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use Regular Expressions</string>
   </property>
  </action>
  <action name="checkBoxSearchOnlyHome">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Search Only Home</string>
   </property>
  </action>
  <action name="checkBoxShowFullPath">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Full Paths</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
